src/codecs/webp/vp8.rs

```diff
@@ -2660,19 +2707,26 @@ mod test {
                 for k in 0u8..=255 {
                     let floor_avg = ((i as f32) + 2.0 * (j as f32) + { k as f32 } + 2.0) / 4.0;
                     let floor_avg = floor_avg.floor() as u8;
-                    assert_eq!(floor_avg, avg3(i, j, k), "avg3({}, {}, {}), expected {}, got {}.", i, j, k, floor_avg, avg3(i, j, k));
+                    assert_eq!(
+                        floor_avg,
+                        avg3(i, j, k),
+                        "avg3({}, {}, {}), expected {}, got {}.",
+                        i,
+                        j,
+                        k,
+                        floor_avg,
+                        avg3(i, j, k)
+                    );
                 }
             }
         }
     }
 
     #[test]
     fn test_edge_pixels() {
```

Comment:
Needs some skip for these, I think, to preserve the structure of prediction inputs.